extensions/positron-python/package.json

```
@@ -1485,6 +1491,12 @@
                     "title": "%python.command.python.execInConsole.title%",
                     "when": "resourceLangId == python && !isInDiffEditor && !virtualWorkspace && shellExecutionSupported"
                 },
+                {
+                    "command": "python.execAppInTerminal",
+                    "group": "navigation@1",
+                    "title": "%python.command.python.execAppInTerminal.title%",
+                    "when": "resourceLangId == python && !isInDiffEditor && !virtualWorkspace && shellExecutionSupported && pythonFileContainsApp"
```

we can add a feature flag or just update this until everything is hooked up?
```suggestion
                    "when": "false"
```

---

Seems nice and simple to hide it as you suggested instead of using a feature flag. I can add your original when clause in my PR.

---

I don't think `pythonFileContainsApp` is set anymore, should these `when` clauses all be updated?